### ISSUE TITLE
Cannot visualize

### ISSUE THREAD

**Issue by mpkuse** [agasy18/caffe-visualizer#1]
Hi,

I was trying to use your tool to visualize caffe network. http://mikkyang.com/project/caffe-visualizer/. The tool doesn't seem to output anything. I am using Chrome Version 46.0.2490.80 (64-bit) to be exact. Layout issues may be? 

I am pasting my net here : 
name: "CIFAR10_quick"
[layer {
  name: "cifar"
  type: "ImageData"
  top: "data"
  top: "label"
  include {
    phase: TRAIN
  }
# transform_param {
# mean_file: "./cifar10/mean.binaryproto"
# }
# data_param {
# source: "./cifar10/cifar10_train_lmdb"
# batch_size: 100
# backend: LMDB
# }

  image_data_param {
    #source: "./data_batch_2.bin_dir/annotation.txt"
    source: "./all_anno.txt"
    batch_size: 50
    new_height: 32
    new_width: 32
}
}
layer {
  name: "cifar"
  type: "ImageData"
  top: "data"
  top: "label"
  include {
    phase: TEST
  }
# transform_param {
# mean_file: "./cifar10/mean.binaryproto"
# }
# data_param {
# source: "./cifar10/cifar10_test_lmdb"
# batch_size: 10
# backend: LMDB
# }

  image_data_param {
    source: "./test_batch.bin_dir/annotation.txt"
    batch_size: 50
    new_height: 32
    new_width: 32
}
}
###### ############ L1

layer {
  name: "conv1"
  type: "Convolution"
  bottom: "data"
  top: "conv1"
  param {
    lr_mult: 1
  }
  param {
    lr_mult: 2
  }
  convolution_param {
    num_output: 32
    pad: 2
    kernel_size: 5
    stride: 1
    weight_filler {
      type: "gaussian"
      std: 0.0001
    }
    bias_filler {
      type: "constant"
    }
  }
}
layer {
  name: "pool1"
  type: "Pooling"
  bottom: "conv1"
  top: "pool1"
  pooling_param {
    pool: MAX
    kernel_size: 3
    stride: 2
  }
}
layer {
  name: "relu1"
  type: "ReLU"
  bottom: "pool1"
  top: "pool1"
}
###### ############### L2

layer {
  name: "conv2"
  type: "Convolution"
  bottom: "pool1"
  top: "conv2"
  param {
    lr_mult: 1
  }
  param {
    lr_mult: 2
  }
  convolution_param {
    num_output: 32
    pad: 2
    kernel_size: 5
    stride: 1
    weight_filler {
      type: "gaussian"
      std: 0.01
    }
    bias_filler {
      type: "constant"
    }
  }
}
layer {
  name: "relu2"
  type: "ReLU"
  bottom: "conv2"
  top: "conv2"
}
layer {
  name: "pool2"
  type: "Pooling"
  bottom: "conv2"
  top: "pool2"
  pooling_param {
    pool: AVE
    kernel_size: 3
    stride: 2
  }
}
###### ######################## L3

layer {
  name: "conv3"
  type: "Convolution"
  bottom: "pool2"
  top: "conv3"
  param {
    lr_mult: 1
  }
  param {
    lr_mult: 2
  }
  convolution_param {
    num_output: 64
    pad: 2
    kernel_size: 5
    stride: 1
    weight_filler {
      type: "gaussian"
      std: 0.01
    }
    bias_filler {
      type: "constant"
    }
  }
}
layer {
  name: "relu3"
  type: "ReLU"
  bottom: "conv3"
  top: "conv3"
}
layer {
  name: "pool3"
  type: "Pooling"
  bottom: "conv3"
  top: "pool3"
  pooling_param {
    pool: AVE
    kernel_size: 3
    stride: 2
  }
}
###### ####### FC1

layer {
  name: "ip1"
  type: "InnerProduct"
  bottom: "pool3"
  top: "ip1"
  param {
    lr_mult: 1
  }
  param {
    lr_mult: 2
  }
  inner_product_param {
    num_output: 64
    weight_filler {
      type: "gaussian"
      std: 0.1
    }
    bias_filler {
      type: "constant"
    }
  }
}
###### ######### FC2

layer {
  name: "ip2"
  type: "InnerProduct"
  bottom: "ip1"
  top: "ip2"
  param {
    lr_mult: 1
  }
  param {
    lr_mult: 2
  }
  inner_product_param {
    num_output: 10
    weight_filler {
      type: "gaussian"
      std: 0.1
    }
    bias_filler {
      type: "constant"
    }
  }
}
###### ######## Accuracy layer

layer {
  name: "accuracy"
  type: "Accuracy"
  bottom: "ip2"
  bottom: "label"
  top: "accuracy"
  include {
    phase: TEST
  }
}
###### ####### Loss layer

layer {
  name: "loss"
  type: "SoftmaxWithLoss"
  bottom: "ip2"
  bottom: "label"
  top: "loss"
}](url)
